src/test/users/index.test.js

```diff
@@ -0,0 +1,60 @@
+import chaihhtp from 'chai-http';
```

Comment:
Pls rename this import @albert85 to avoid confusion

Reply:
done